client/post-editor/post-editor.jsx

```diff
@@ -713,9 +713,6 @@ export const PostEditor = React.createClass( {
 			this.props.resetEditorLastDraft();
 		}
 
-		// Remove this when the editor is completely reduxified ( When using Redux actions for all post saving requests )
-		this.props.savePostSuccess( post.site_ID, this.props.postId, post, {} );
```

Comment:
This was introduced here https://github.com/Automattic/wp-calypso/pull/10177 and the idea IIRC was to update the editedPostId in the state when we're creating a new post. If we drop this, the editedPostId on the state will not reflect the saved post id. I'm not sure What's the impact yet.